modules/planning/tasks/traffic_decider/crosswalk.cc

```diff
@@ -184,11 +212,34 @@ void Crosswalk::MakeDecisions(Frame* const frame,
         stop = true;
       }
 
+      if (stop) {
+        const double kMaxStopSpeed = 0.3;
+        auto obstacle_speed = std::hypot(perception_obstacle.velocity().x(),
+                                         perception_obstacle.velocity().y());
+        if (stopped_at_crosswalk && obstacle_speed <= kMaxStopSpeed) {
+          // check on stop timer
+          if (crosswalk_stop_timer.first == crosswalk_overlap->object_id) {
+            auto stop_timers = crosswalk_stop_timer.second;
```

Comment:
use `const auto& stop_timers`

Reply:
it is not a const.  it's a local variable and can be updated.